app/serializers/floof_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class FloofSerializer < ActiveModel::Serializer
+  attributes :id, :name, :job_title, :current_employer, :species, :category
+
+  def category
+    object.category.name
+  end
```

Comment:
Do you still need to serve up the category name this way? Are you using it somewhere? If not, I'd suggest removing this!